src/Collection.php

```diff
@@ -927,6 +931,8 @@ public function listSearchIndexes(array $options = []): Iterator
      */
     public function mapReduce(JavascriptInterface $map, JavascriptInterface $reduce, $out, array $options = [])
     {
+        @trigger_error(sprintf('The %s method is deprecated and will be removed in a future release.', __METHOD__), E_USER_DEPRECATED);
```

Comment:
Will you remove it in version 2.0? In that case, you can mention the version here.
```suggestion
        @trigger_error(sprintf('The %s method is deprecated and will be removed in version 2.0.', __METHOD__), E_USER_DEPRECATED);
```

Reply:
Updated all occurrences of "in a future release" to refer to 2.0.